### PR TITLE
Add constraint name validation and escaping

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: lib/pg.ddl.js [hunk 1]
```diff
@@ -301,7 +301,10 @@ const generateIndexes = (indexes, table) => indexes
       index.fields.map(escapeIdentifier).join(', ') :
       escapeIdentifier(name);
 
-    return `CREATE INDEX idx${table}${name} ` +
+    const indexName = `idx${table}${name}`;
+    validateIdentifier(name, 'index');
+
+    return `CREATE INDEX  ${escapeIdentifier(indexName)} ` +
```
Comment (belochub) [inline lib/pg.ddl.js]: Redundant space after 'INDEX' here.

File: lib/pg.ddl.js [hunk 2]
```diff
@@ -391,9 +403,13 @@ const generateTable = (
...
     getResolvableLinks(name, unresolvedLinks, links, existingTables);
 
+  const pk = `pk${name}Id`;
+  validateIdentifier(pk, 'constraint');
+
   const primaryKey =
     `ALTER TABLE ${escapeIdentifier(name)} ` +
-    `ADD CONSTRAINT pk${name}Id PRIMARY KEY (${escapeIdentifier('Id')});`;
+    `ADD CONSTRAINT ${escapeIdentifier(pk)} ` +
+    `PRIMARY KEY (${escapeIdentifier('Id')});`;
```
Comment (belochub) [inline lib/pg.ddl.js]: Why is `'Id'` escaped here using `escapeIdentifier`? It is not escaped on line
https://github.com/metarhia/globalstorage/blob/1386aced2697ab0c859808f9c808aacd1ac61d07/lib/pg.ddl.js#L231

Comment (nechaido) [reply lib/pg.ddl.js]: It is now escaped.

**Comment by nechaido** [comment]
Landed in b62877e.

**Pull request closed without merging**